springfox-swagger-common/build.gradle

```diff
@@ -16,18 +16,25 @@ dependencies {
   compile project(':springfox-spring-web')
   compile libs.core
   compile libs.spring
+  compile group: 'org.springframework.boot', name: 'spring-boot-autoconfigure', version: springBoot
 
   provided libs.springProvided
   provided libs.clientProvided
   provided libs.springProvidedWithMvc
+  provided group: 'io.projectreactor', name: 'reactor-core', version: '3.2.6.RELEASE'
 
   testCompile project(':springfox-core')
   testCompile project(':springfox-core').sourceSets.test.output
   testCompile project(':springfox-schema').sourceSets.test.output
   testCompile project(':springfox-spring-web').sourceSets.test.output
   testCompile project(':springfox-spring-webmvc').sourceSets.test.output
   testCompile project(':springfox-spring-webmvc')
+  testCompile project(':springfox-spring-webflux')
 
-  testCompile libs.test
+  testCompile group: 'org.powermock', name: 'powermock-api-mockito2', version: '2.0.0'
+  testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: '2.0.0'
```

Comment:
Why power mock vs. mockito

Reply:
The power mock is the only way that I could find to mock static methods, which is needed by the util class `ClassUtils` (*containing three static methods*).

Reply:
Maybe keep it unresolved?